src/components/Superscript.native.tsx

```diff
@@ -65,13 +66,12 @@ const Superscript: FC<SuperscriptProps> = ({ empty, numContexts, showSingle, sup
   //   .reduce((charCount, child) => charCount + child.length, 0)
```

Comment:
are we planning to use this commented out code?

Reply:
removed